src/XML/src/FMI3/fmi3_xml_parser.c

```diff
@@ -29,6 +29,7 @@
 #include "../FMI/fmi_xml_terminals_and_icons_impl.h"
 #include "fmi3_xml_parser.h"
 #include "JM/jm_portability.h"
+#include "fmi3_xml_parser_context_impl.h" // TODO: Really needed here?
```

Comment:
Is the comment still relevant?

Reply:
Yes, somewhat, will be dealt with in follow-up cleanup.